plugins/com.google.cloud.tools.eclipse.appengine.newproject.test/src/com/google/cloud/tools/eclipse/appengine/newproject/CreateAppEngineStandardWtpProjectTest.java

```diff
@@ -52,41 +60,77 @@
   private NullProgressMonitor monitor = new NullProgressMonitor();
   private AppEngineStandardProjectConfig config = new AppEngineStandardProjectConfig();
   private IProject project;
-  
+
   @Before
   public void setUp() {
     IWorkspace workspace = ResourcesPlugin.getWorkspace();
     project = workspace.getRoot().getProject("testproject" + Math.random());
     config.setProject(project);
   }
-  
+
   @After
   public void cleanUp() throws CoreException {
     project.delete(true, monitor);
   }
-  
+
+  // TODO(chanseok): Debugging code. Remove after the issue is resolved:
+  // https://github.com/GoogleCloudPlatform/google-cloud-eclipse/issues/1086
+  // (Don't forget to remove Guava from MANIFEST.MF.)
+  private void logForSetPrimaryRuntimeError() {
+    assertTrue(project == config.getProject());
```

Comment:
Is it your intention to fail the test if this is not true?
assertEquals or assertSame, whichever you need